src/Compilers/CSharp/Test/Semantic/Semantics/BindingAsyncTests.cs

```diff
@@ -1214,10 +1214,14 @@ interface IInterface
 {
     async void F(); 
 }";
-            CreateCompilationWithMscorlib45(source).VerifyDiagnostics(
-                // (4,16): error CS0106: The modifier 'async' is not valid for this item
+            CreateCompilationWithMscorlib45(source, parseOptions:TestOptions.Regular7).VerifyDiagnostics(
+                // (4,16): error CS8503: The modifier 'async' is not valid for this item in C# 7. Please use language version 7.1 or greater.
```

Comment:
I don't think we can put 8.0 in master (that is a public API).

---
In reply to: [110452527](https://github.com/dotnet/roslyn/pull/18519#discussion_r110452527) [](ancestors = 110452527)